lib/jekyll-import/importers/joomla.rb

```diff
@@ -32,11 +33,12 @@ def self.process(options)
         dbname  = options.fetch('dbname')
         user    = options.fetch('user')
         pass    = options.fetch('password', '')
-        host    = options.fetch('host', "localhost")
+        host    = options.fetch('host', 'localhost')
+        port    = options.fetch('port', '3306')
```

Comment:
Hmm, looks [like](https://github.com/brianmario/mysql2/blob/master/lib/mysql2/client.rb#L85) [it](https://github.com/brianmario/mysql2/blob/master/ext/mysql2/client.c#L78). Should I add `to_i` or change the default value... or both?